status_code.gemspec

```diff
@@ -0,0 +1,22 @@
+Gem::Specification.new do |s|
+  s.name        = 'status_code'
+  s.version     = '0.0.1'
+  s.date        = '2017-01-20'
+  s.summary     = 'Decodes credit card transaction status code'
+  s.description = 'Returns decoding credit card transaction status code' \
```

Comment:
Maybe, `Responsible for decoding transactions result codes generated by various acquirer banks`. So, I propose: `Decoding transaction bank codes` for summary